actions/status-checker/src/pull-updater.ts

```diff
@@ -30,8 +30,13 @@ export async function tryUpdatePullRequestBody(token: string) {
       return;
     } else {
       try {
+        console.log("::group::Pull request JSON body");
```

Comment:
Probably best to future proof this by instead adding `core` to the existing `import { context, getOctokit } from "@actions/github";`, and then calling:

```ts
core.startGroup('Pull request JSON body');
core.info(JSON.stringify(pullRequest, undefined, 2));
core.endGroup();
```

See https://github.com/actions/toolkit/tree/main/packages/core#logging